src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -92,7 +96,14 @@ method newMessagesLoaded*(self: Module, messages: seq[MessageDto], reactions: se
 
     for r in reactions:
       if(r.messageId == m.id):
-        item.addReaction(r.emojiId, m.`from`, r.id)
+        var emojiIdAsEnum: EmojiId
+        if(message_reaction_item.toEmojiIdAsEnum(r.emojiId, emojiIdAsEnum)):
+          let userWhoAddedThisReaction = self.controller.getContactById(r.`from`)
+          let amIReactedWithThisEmoji = userWhoAddedThisReaction.id == singletonInstance.userProfile.getPubKey()
```

Comment:
Not a big deal, but this name isn't grammatically correct
It should be either `amITheOneWhoReactedWithThisEmoji`
or `didIReactWithThisEmoji`
Worst case, `isCurrentUser` works as well